data-upgrade-ecms/src/main/resources/conf/portal/configuration.xml

```diff
@@ -82,6 +82,40 @@
         </value-param>
       </init-params>
     </component-plugin>
+
+    <component-plugin>
+      <name>DocumentPermissionUpgradePlugin</name>
+      <set-method>addUpgradePlugin</set-method>
+      <type>org.exoplatform.ecms.upgrade.templates.DocumentPermissionUpgradePlugin</type>
+      <description>Upgrade templates for node types</description>
+      <init-params>
+        <value-param>
+          <name>product.group.id</name>
+          <description>The groupId of the product</description>
+          <value>org.exoplatform.ecms</value>
+        </value-param>
+        <value-param>
+          <name>plugin.execution.order</name>
+          <description>The plugin execution order</description>
+          <value>3</value>
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.execute.once</name>
+          <description>Execute this upgrade plugin only once</description>
+          <value>true</value>
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.async.execution</name>
+          <description>Execute this upgrade asynchronously</description>
+          <value>true</value>
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.target.version</name>
+          <description>Target version of the plugin</description>
+          <value>6.2.0</value>
```

Comment:
the plugin will only be triggered for 6.2.0 version ? If so, how are we going to deal with continuous delivery and milestone deploymnents?